test/index.ts

```diff
@@ -72,34 +90,95 @@ describe('Mutation Test', () => {
             }
           }
         `,
-        variables: { credentials: testUser },
+        variables: { credentials: correctInputUser },
       },
     });
 
-    expect(createUseMutation.data.data.createUser.email).to.be.eq(testUser.email);
-    expect(createUseMutation.data.data.createUser.name).to.be.eq(testUser.name);
-    expect(createUseMutation.data.data.createUser.birthDate).to.be.eq(testUser.birthDate);
+    const mutationReturn = createUseMutation.data.data.createUser;
+
+    expect({
+      email: mutationReturn.email,
+      name: mutationReturn.name,
+      birthDate: mutationReturn.birthDate,
+    }).to.be.deep.eq({
+      email: correctInputUser.email,
+      name: correctInputUser.name,
+      birthDate: correctInputUser.birthDate,
+    });
     expect(createUseMutation.data.data.createUser.id).to.exist;
-  });
 
-  it('Is user included in database?', async () => {
-    const testUserFromDatabase = await AppDataSource.manager.findOneBy(User, { email: testUser.email });
+    const testUserFromDatabase = await AppDataSource.manager.findOneBy(User, { email: correctInputUser.email });
 
-    const testUserHashedPasword = generateHashPasswordFromSalt(testUserFromDatabase.salt, testUser.password);
-    delete testUserFromDatabase.id;
-    delete testUserFromDatabase.salt;
+    expect(Number.isInteger(testUserFromDatabase.id)).to.be.true;
 
+    const testUserHashedPasword = generateHashPasswordFromSalt(testUserFromDatabase.salt, correctInputUser.password);
+    delete testUserFromDatabase.salt;
+    delete testUserFromDatabase.id;
     expect(testUserFromDatabase).to.be.deep.eq({
-      ...testUser,
+      ...correctInputUser,
       password: testUserHashedPasword,
     });
   });
 
-  it('User has been removed', async () => {
-    await AppDataSource.manager.delete(User, { email: testUser.email });
+  it('Email Already Exists Mutation', async () => {
+    const createUseMutation = await axios({
+      url,
+      method: 'post',
+      data: {
+        query: `
+          mutation CreateUser($credentials: UserInput!) {
+            createUser(data: $credentials) {
+              id,
+              name,
+              email,
+              birthDate
+            }
+          }
+        `,
+        variables: { credentials: correctInputUser },
+      },
+    });
+
+    const emailError = {
+      message: errorsMessages.existingEmail,
+      code: 400,
+      additionalInfo: null,
+    };
+
+    const errors = createUseMutation.data.errors;
+
+    expect(errors).to.be.an('array').that.deep.includes(emailError);
```

Comment:
```suggestion
    expect(errors).to.be.deep.eq([emailError]);
```

Assim funciona tbm né?